gnuradio-runtime/lib/math/qa_sincos.cc

```diff
@@ -24,14 +24,11 @@
 #include <config.h>
 #endif
 
-#include <qa_sincos.h>
 #include <gnuradio/sincos.h>
-#include <cppunit/TestAssert.h>
+#include <boost/test/unit_test.hpp>
 #include <cmath>
 
-void
-qa_sincos::t1()
-{
+BOOST_AUTO_TEST_CASE(t1) {
```

Comment:
General question: We could have cooler names for test cases with Boost, right? (not a request for change)